src/eos/radiationProperties/radiationZimmer.hpp

```diff
@@ -0,0 +1,70 @@
+#ifndef ABLATELIBRARY_RADIATIONPROPERTIESZIMMER_H
+#define ABLATELIBRARY_RADIATIONPROPERTIESZIMMER_H
+
+#include "eos/radiationProperties/radiationProperties.hpp"
+#include "finiteVolume/compressibleFlowFields.hpp"
+#include "radiationProperties.hpp"
+#include "solver/cellSolver.hpp"
+#include "utilities/mathUtilities.hpp"
+
+namespace ablate::eos::radiationProperties {
+/** A radiation gas absorption model which computes the absorptivity based on the presence of certain species. */
+class Zimmer : public RadiationModel {
```

Comment:
Could we rename the radaitionZimmer.hpp and cpp files to match the name of the class?